website/docs/spec/registry.md

```diff
@@ -122,22 +130,22 @@ The `omgidl` schema encoding stores [OMG IDL](https://www.omg.org/spec/IDL/4.2/A
 
 #### Producing `schema.data` for the `omgidl` schema encoding
 
-`.idl` files usually contain only one definition each, and use C preprocessor
+`.idl` files usually contain only one definition each, and use C++ preprocessor
 `#include` directives when referring to other definitions. However, the `data`
 field must be the text of a single, self-contained OMG IDL source file. That is,
 all referenced type definitions must be present, and there must be no
 preprocessor directives.
 
-You can produce the required format with a C preprocessor. For example, with GCC
+You can produce the required format with a C++ preprocessor. For example, with GCC
 or Clang:
 
 ```bash
-# -x c: treat input as C to avoid guessing based on the extension.
+# -x c++: treat input as C++ to avoid guessing based on the extension.
 # -E: run only the preprocessor.
 # -P: remove all #-directives from the output.
 # Run from the root of your IDL definition tree, or change the `-I` flag to the
 # correct directory.
-$ cc -x c -E -P -I . top_level_module/my_module/MyType.idl > combined.idl
+$ cc -x c++ -E -P -I . top_level_module/my_module/MyType.idl > combined.idl
```

Comment:
I don't have any particular objection to treating these includes as C++ rather than C, I'm just curious why the change was made?